index.html

```diff
@@ -54,15 +54,12 @@ <h1>Welcome to Malcolm’s CSD120 Website!</h1>
               "
               alt="A mugshot of Malcolm"
             />
-          </a>
-          <div class="paperclip">
             <img
               class="paperclip"
               src="Images/paperclip.png"
               alt="An image paperclip"
             />
-          </div>
-
+          </a>
```

Comment:
The paperclip was not to be in a div of its own; it was to be the last child of the selfie hyperlink